docs/masking-providers.md

```diff
@@ -733,6 +734,36 @@ date at */dateWritten*.
 | preserveDomains | Integer  | Number of domains to preserve starting from the right; if value = -1 then all domains will be preserved (that is, everything after \@) | 1                 |
 | nameLength      | Integer  | Length of username to generate; if value = -1, then the username will be 5 to 8 characters long. 0 is invalid      | \-1               |
 
+#### FHIR_MORTALITY_DEPENDENCY
+
+This provider enforces a specific type of dependency between the birth date and the deceased datetime and deceased boolean properties that exist in a FHIR Patient resource, which is a very specific schema imposed on a JSON document. 
+
+This provider may only be applied to properties named either exactly "deceasedDateTime" or "deceasedBoolean".  When using this provider, it is recommended to configure the provider on both of these properties. It is expected that both of these properties and another property named "birthDate" all belong to the same parent object in the JSON document. 
+
+When the privacy provider is applied to "deceasedDateTime" and "deceasedDateTime" has a non-null value in the input document, the value of "deceasedDateTime" is set to null.  In addition, the value of "deceasedBoolean" is either set to _true_ or _null_ depending upon the age of this patient when death occurred.  If "deceasedBoolean" does not exist as a sibling of "deceasedDateTime", it will be added to the JSON structure. 
```

Comment:
"depending upon the age of this patient when death occurred". 

At what age is deceasedBoolean set to _true_?

Reply:
The following paragraphs explain about the age threshold - basically its a configurable number of years.  If you are older than that, you can have deceasedBoolean=true.  If not, you get deceasedBoolean=null.